bin/rspecq

```diff
@@ -95,19 +107,22 @@ opts[:max_requeues] ||= Integer(ENV["RSPECQ_MAX_REQUEUES"] || DEFAULT_MAX_REQUEU
 raise OptionParser::MissingArgument.new(:build) if opts[:build].nil?
 raise OptionParser::MissingArgument.new(:worker) if !opts[:report] && opts[:worker].nil?
 
+redis_opts = {host: opts[:redis_host]}
```

Comment:
We like to declare hashes in multiple lines for readability:

```ruby
redis_opts = {
  host: opts[:redis_host]
}
```

Reply:
```suggestion
redis_opts = {
  host: opts[:redis_host]
}
```